src/models/Variant.php

```diff
@@ -68,7 +68,8 @@ public function getVat()
      */
     public function thumbnailImage()
     {
-        return \dmstr\willnorrisImageproxy\Url::image($this->thumbnail_image);
+        $preset = \Yii::$app->settings->get('imagePreset', 'shop');
```

Comment:
Kannst du als dritten Parameter (default) dem getter bitte ein leerzeichen mitgeben, damit funktioniert der Url helper dann wie davor weil `'' !== null`

Reply:
Added default value for get as requested